DESCRIPTION

```diff
@@ -1,14 +1,14 @@
 Package: drat
 Type: Package
 Title: 'Drat' R Archive Template
-Version: 0.1.6
-Date: 2020-05-29
+Version: 0.2.0
```

Comment:
Please leave changes to version and dates to the release manager aka maintainer.  Thank you.